### PR TITLE
Haaglumination

### PR DESCRIPTION
**Pull request by MHaggis**
New Analytic Story
- Lumma Stealer

New Analytics
- Windows RunMRU Command Execution
- Windows BitLockerToGo Process Execution
- Windows BitLockerToGo with Network Activity

Tagged content
 - Malicious PowerShell Process - Encoded Command
 - PowerShell 4104 Hunting
 - Suspicious mshta child process
 - Detect MSHTA Url in Command Line

### CODE REVIEW

**Comment by mvelazc0** [comment]
love it

File: detections/endpoint/windows_bitlockertogo_with_network_activity.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+how_to_implement: To successfully implement this search, you need to be ingesting logs with the process name and eventcode = 22 dnsquery executions from your endpoints. If you are using Sysmon, you must have at least version 6.0.4 of the Sysmon TA. 
+known_false_positives: False positives are likely, as BitLockerToGo.exe is a legitimate Windows utility used for managing BitLocker encryption. However, the detection is designed to flag unusual execution patterns that deviate from standard usage. Filtering may be required to reduce false positives, once confirmed - move to TTP.
+references:
+- https://any.run/report/5e9ba24639f70787e56f10a241271ae819ef9c573edb22b9eeade7cb40a2df2a/66f16c7b-2cfc-40c5-91cc-f1cbe9743fa3
+- https://securelist.com/fake-captcha-delivers-lumma-amadey/114312/
+drilldown_searches:
```
Comment (patel-bhavin) [inline detections/endpoint/windows_bitlockertogo_with_network_activity.yml]: @MHaggis : you can remove drilldowns for the two Hunting searches

File: detections/endpoint/windows_bitlockertogo_process_execution.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+how_to_implement: The detection is based on data that originates from Endpoint Detection and Response (EDR) agents. These agents are designed to provide security-related telemetry from the endpoints where the agent is installed. To implement this search, you must ingest logs that contain the process GUID, process name, and parent process. Additionally, you must ingest complete command-line executions. These logs must be processed using the appropriate Splunk Technology Add-ons that are specific to the EDR product. The logs must also be mapped to the `Processes` node of the `Endpoint` data model. Use the Splunk Common Information Model (CIM) to normalize the field names and speed up the data modeling process.
+known_false_positives: False positives are likely, as BitLockerToGo.exe is a legitimate Windows utility used for managing BitLocker encryption. However, monitor for usage of BitLockerToGo.exe in your environment, tune as needed. If BitLockerToGo.exe is not used in your environment, move to TTP.
+references:
+- https://securelist.com/fake-captcha-delivers-lumma-amadey/114312/
+drilldown_searches:
+- name: View the detection results for - "$dest$"
```
Comment (patel-bhavin) [inline detections/endpoint/windows_bitlockertogo_process_execution.yml]: you can remove drilldowns for the two Hunting searches

File: detections/endpoint/windows_bitlockertogo_process_execution.yml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  message: BitLockerToGo.exe was executed on $dest$ by $user$.
+  mitre_attack_id:
+  - T1218
+  observable:
+  - name: dest
+    type: Endpoint
+    role:
+    - Victim
+  - name: user
+    type: User Name
```
Comment (patel-bhavin) [inline detections/endpoint/windows_bitlockertogo_process_execution.yml]: can we change this type to User in all the new detections? we recently cleaned up all our yamls to make it consitent. We are yet to enforce this in contentctl!

**Review by patel-bhavin — CHANGES_REQUESTED** [review]

**Review by patel-bhavin — DISMISSED** [review]

**Review by patel-bhavin — APPROVED** [review]

**Pull request merged**